compiler/rustc_codegen_llvm/src/builder.rs

```diff
@@ -225,17 +226,21 @@ impl<'a, 'll, 'tcx> BuilderMethods<'a, 'tcx> for Builder<'a, 'll, 'tcx> {
         let args = self.check_call("invoke", llty, llfn, args);
         let funclet_bundle = funclet.map(|funclet| funclet.bundle());
         let funclet_bundle = funclet_bundle.as_ref().map(|b| &*b.raw);
-        let mut bundles = vec![funclet_bundle];
+        let mut bundles: SmallVec<[_; 2]> = SmallVec::new();
+        if funclet_bundle.is_some() {
+            bundles.push(funclet_bundle);
```

Comment:
maybe we could stop putting options in there and just use the value directly in LLVMRustBuildCall?

Reply:
🤣 I should have looked at the next commit